lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -36,7 +36,11 @@ module.exports = {
       params.NotificationARNs = this.serverless.service.provider.notificationArns;
     }
 
-    return this.createDeployment().then(() => this.provider.request(
+    const deploymentData = {
+      provider: this.serverless.service.provider.name,
+      region: this.serverless.service.provider.region,
+    };
+    return this.createDeployment(deploymentData).then(() => this.provider.request(
```

Comment:
hmmm that doesn't seem right! the `createDeployment` call does not take any inputs!